Configure.ps1

```diff
@@ -4,8 +4,8 @@
 # Before running this script you need to install the Azure RM cmdlets as an administrator. 
```

Comment:
You should add a line to import the correct PowerShell module. That way the script will fail upfront if the module is not found.